python/pyspark/sql/dataframe.py

```diff
@@ -2153,6 +2153,22 @@ def transform(self, func):
                                               "should have been DataFrame." % type(result)
         return result
 
+    @since(3.0)
```

Comment:
It's during the code freeze and RC will start next week. Is there a reason to target Spark 3.0? cc @mengxr.

Reply:
@HyukjinKwon We can change this as develop API.

Reply:
We don't need it for 3.0.